### DISCUSSION TITLE
Just have to check: Did **anyone** get any hits during the last few days (e.g., Black Friday or Cyber Mon.)?

### DISCUSSION THREAD

**Discussion by Ben7530** [jef/streetmerchant#1043] [Help]
### Description

 I got nothing at any point. Makes me wonder if my setup is correct. 

I had a wild idea that maybe companies were holding back stock until cyber monday, but NOPE! :(

**Comment by huntinganswers** [comment]
no.. nobody has released any in the states other than adorama ... the bot doesn't work with the website for whatever reason.

..the playstation store did, but there was a que

**Comment by hectic-droid** [comment]
I did get a couple of hits on Amazon and Target for the PS5 but were sold out before I got to them.  The one on Amazon was used.  I also got 77 hits for the Xbox Series S/X from Best Buy on Nov 28.  Was able to order 2 S and 1 X.

**Comment by huntinganswers** [comment]
target didn't even ding for me this morning.

**Comment by hectic-droid** [comment]
> target didn't even ding for me this morning.

Yeah, I’m following Wario64 on twitter.  That’s how I found out.  Had to search for stores outside my area but found one 75 miles north of my home.  Was going there to do service on my vehicle today anyways.  Just received the order ready email from Target.

My bot did not alert me as they never went in stock in the store near me.  You also wanna check popfindr to see which target has inventory, make that store your store and then refresh often until they launch them.

**Comment by jef** [comment]
I've gotten confirmation with people that they've gotten ryzen chips and playstations. So it really depends.
